features/246-custom-domain/index.md

```diff
@@ -76,6 +76,9 @@ Allow developer to setup custom domain for their application, basically followin
 - `A` record is used for setting up DNS record, so both apex domain and subdomain can be supported.
 - After developer verified a domain, Skygear Cluster will verify the root domain by checking `TXT` record. Once the root domain is verified, the root domain will be added to that specific app. Developer will not need to add `TXT` record again for the new subdomains in that app, only `A` record will be checked.
 - Configure custom domain requires root domain ownership verification, so it is not supported if developer can control the subdomain only (e.g. free domain).
+- If developer wants to use CDN with custom domain, the CDN DNS records may be used instead of the provided
+`A` record. In this case, developer can pass a flag to the verification command to skip verification of
```

Comment:
I guess this should be an persistent attribute of custom domain like `redirect_domain` instead of a flag of the verify command. When we later have periodic verification, we will not know whether A record should be skipped.